generators/client/templates/angular/src/main/webapp/app/admin/audits/audits.component.ts.ejs

```diff
@@ -30,113 +29,109 @@ import { AuditsService } from './audits.service';
   selector: '<%= jhiPrefixDashed %>-audit',
   templateUrl: './audits.component.html'
 })
-export class AuditsComponent implements OnInit, OnDestroy {
-    audits: Audit[];
-    fromDate: string;
-    itemsPerPage: any;
-    links: any;
-    page: number;
-    routeData: any;
-    predicate: any;
-    previousPage: any;
-    reverse: boolean;
-    toDate: string;
-    totalItems: number;
+export class AuditsComponent implements OnInit {
+    private dateFormat = 'yyyy-MM-dd';
```

Comment:
I would put private fields at the end of public fields

Reply:
Done.